src/execution/mod.rs

```diff
@@ -328,63 +450,6 @@ where
     }
 }
 
-/// Compile a dataflow which reads the progress data from the previous
-/// execution and calculates the resume epoch.
-fn build_resume_epoch_calc_dataflow<A: Allocate>(
```

Comment:
Moved into `crate::recovery` so that all the reading and writing logic for state is in one spot.